### ISSUE TITLE
It can't lemmatize word `goes`

### ISSUE THREAD

**Issue by nukuutos** [winkjs/wink-lemmatizer#31]
Current behavior:
```js
lemmatize.verb('goes') // -> goes
```
Expected:
```js
lemmatize.verb('goes') // -> go
```

**Comment by sanjayaksaxena** [comment]
Thanks @nukuutos for highlighting the issue. Will release the fix shortly.

Best,
Sanjaya

**Comment by sanjayaksaxena** [comment]
Hi @nukuutos,

Have released a version 3.0.3 with a temp fix. Consider using [winkNLP](https://github.com/winkjs/wink-nlp) if you need comprehensive NLP pipe. 

Best,
Sanjaya
